query_generators/anthropic.py

```diff
@@ -7,10 +7,32 @@
 from query_generators.query_generator import QueryGenerator
 from utils.pruning import prune_metadata_str
 from utils.gen_prompt import to_prompt_schema
+from utils.dialects import (
+    ddl_to_bigquery,
+    ddl_to_mysql,
+    ddl_to_sqlite,
+    ddl_to_tsql,
+)
 
 anthropic = Anthropic(api_key=os.environ.get("ANTHROPIC_API_KEY"))
 
 
+def convert_ddl(postgres_ddl: str, to_dialect: str, db_name: str):
+    if to_dialect == "postgres":
+        return postgres_ddl
+    elif to_dialect == "bigquery":
+        new_ddl, _ = ddl_to_bigquery(postgres_ddl, "postgres", db_name, 42)
+    elif to_dialect == "mysql":
+        new_ddl, _ = ddl_to_mysql(postgres_ddl, "postgres", db_name, 42)
+    elif to_dialect == "sqlite":
+        new_ddl, _ = ddl_to_sqlite(postgres_ddl, "postgres", db_name, 42)
+    elif to_dialect == "tsql":
+        new_ddl, _ = ddl_to_tsql(postgres_ddl, "postgres", db_name, 42)
+    else:
+        raise ValueError(f"Unsupported dialect {to_dialect}")
+    return new_ddl
```

Comment:
do you wanna add this to utils.dialect so it doesn't have to be repeated across the files?

Reply:
Will do!